apps/desktop-wallet/src/api/useAddressesDisplayTokens.ts

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { AddressHash } from '@alephium/shared'
+import { orderBy } from 'lodash'
+import { useMemo } from 'react'
+
+import { useAddressesTokensBalances } from '@/api/addressesBalancesDataHooks'
+import { useAddressesListedFungibleTokens } from '@/api/addressesListedFungibleTokensDataHooks'
+import { useAddressesNfts } from '@/api/addressesNftsDataHooks'
+import { useAddressesTokensWorth } from '@/api/addressesTokensPricesDataHooks'
+import {
+  useAddressesUnlistedFungibleTokens,
+  useAddressesUnlistedNonStandardTokenIds
+} from '@/api/addressesUnlistedTokensHooks'
+import { ListedFTDisplay, NFTDisplay, NonStandardTokenDisplay, TokenDisplay, UnlistedFTDisplay } from '@/types/tokens'
+
+const useAddressesDisplayTokens = (addressHash?: AddressHash) => {
+  const { data: tokensBalances, isLoading: isLoadingTokensBalances } = useAddressesTokensBalances(addressHash)
+  const { data: tokensWorth, isLoading: isLoadingTokensWorth } = useAddressesTokensWorth(addressHash)
+  const { data: listedFTs, isLoading: isLoadingListedFTs } = useAddressesListedFungibleTokens(addressHash)
+  const { data: unlistedFTs, isLoading: isLoadingUnlistedFTs } = useAddressesUnlistedFungibleTokens(addressHash)
```

Comment:
Should the hooks use the same naming paradigm? `useAddressesListedFTs`?

Reply:
Sure!